akka-http-core/src/test/scala/akka/http/scaladsl/ClientServerSpec.scala

```diff
@@ -82,8 +82,7 @@ class ClientServerSpec extends AkkaSpecWithMaterializer(
     }
 
     "report failure if bind fails" in EventFilter[BindException](occurrences = 2).intercept {
-      val (hostname, port) = SocketUtil.temporaryServerHostnameAndPort()
-      val binding = Http().newServerAt(hostname, port).connectionSource()
+      val binding = Http().newServerAt("localhost", 0).connectionSource()
```

Comment:
I think we probably need a non-zero port here for this test to work?
I tried passing port = `1` but it gave me `java.net.SocketException: Permission denied` instead of `BindException`.

Reply:
non-root users typically can't bind to port numbers below 1024, so a larger number might work?

Reply:
Thanks, @raboof. It worked.